tsdb/ooo_head_read.go

```diff
@@ -513,7 +513,7 @@ type HeadAndOOOQuerier struct {
 	head       *Head
 	index      IndexReader
 	chunkr     ChunkReader
-	querier    storage.Querier
+	querier    storage.Querier // This might be nil if head was truncated.
```

Comment:
Generally I advise to state what the thing means, not when you expect it to apply.
So "If nil, do not read from in-order head"?

Reply:
clarified